pose_pipeline/env.py

```diff
@@ -1,8 +1,17 @@
 import os
 import sys
+import datajoint as dj
+from pathlib import Path
+from dotenv import load_dotenv
+from .utils.paths import find_full_path
+from .paths import get_pose_project_dir
+
+logger = dj.logger
```

Comment:
do we need this here? might be nicer to let people independently choose if they want to change their system logging.

Reply:
Sure. I'll remove here